### PR TITLE
ticdc: remove canal protocol document

### PR DESCRIPTION
**Pull request by wk989898**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
 Canal protocol is  not officially claimed to be supported.
Screenshot 2024-03-15 at 12 28 06

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Need modification after applied to another branch 
- [x] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-faq.md [hunk 1]
```diff
@@ -110,18 +110,19 @@ If you use the `cdc cli changefeed create` command without specifying the `-conf
 
 ## Does TiCDC support outputting data changes in the Canal format?
 
-Yes. To enable Canal output, specify the protocol as `canal` in the `--sink-uri` parameter. For example:
+Yes. To enable Canal output, specify the protocol as `canal-json` in the `--sink-uri` parameter. For example:
```
Comment (flowbehappy) [inline ticdc/ticdc-faq.md:113]:
```suggestion
Yes. Note that for Canal protocol, TiCDC only support the JSON output format, while the protobuf format is not officially supported yet. To enable Canal output, specify the protocol as `canal-json` in the `--sink-uri` parameter. For example:
```

File: ticdc/ticdc-faq.md [hunk 2]
```diff
@@ -110,18 +110,19 @@ If you use the `cdc cli changefeed create` command without specifying the `-conf
 
 ## Does TiCDC support outputting data changes in the Canal format?
```
Comment (flowbehappy) [inline ticdc/ticdc-faq.md:111]:
```suggestion
## Does TiCDC support outputting data changes in the Canal protocol?
```

File: ticdc/ticdc-faq.md [hunk 3]
```diff
@@ -110,18 +110,19 @@ If you use the `cdc cli changefeed create` command without specifying the `-conf
...
-cdc cli changefeed create --server=http://127.0.0.1:8300 --sink-uri="kafka://127.0.0.1:9092/cdc-test?kafka-version=2.4.0&protocol=canal" --config changefeed.toml
+cdc cli changefeed create --server=http://127.0.0.1:8300 --sink-uri="kafka://127.0.0.1:9092/cdc-test?kafka-version=2.4.0&protocol=canal-json" --config changefeed.toml
 ```
 
 > **Note:**
 >
 > * This feature is introduced in TiCDC 4.0.2.
-> * TiCDC currently supports outputting data changes in the Canal format only to MQ sinks such as Kafka.
+> * TiCDC currently supports outputting data changes in the Canal-JSON format only to MQ sinks such as Kafka.
+> * The difference between Canal and Canal-JSON is that Canal uses protobuf to do the encoding, while Canal-JSON uses JSON.
```
Comment (flowbehappy) [inline ticdc/ticdc-faq.md:125]:
```suggestion
```
We simply don't mention `Canal` at all.

**Review by 3AceShowHand — APPROVED** [review]

File: ticdc/ticdc-faq.md [hunk 4]
```diff
@@ -108,20 +108,20 @@ If you use the `cdc cli changefeed create` command without specifying the `-conf
 - Replicates all tables except system tables
 - Only replicates tables that contain [valid indexes](/ticdc/ticdc-overview.md#best-practices)
 
-## Does TiCDC support outputting data changes in the Canal format?
+## Does TiCDC support outputting data changes in the Canal protocol?
 
-Yes. To enable Canal output, specify the protocol as `canal` in the `--sink-uri` parameter. For example:
+Yes. Note that for Canal protocol, TiCDC only support the JSON output format, while the protobuf format is not officially supported yet. To enable Canal output, specify the protocol as `canal-json` in the `--sink-uri` parameter. For example:
```
Comment (lilin90) [inline ticdc/ticdc-faq.md:113]:
```suggestion
Yes. Note that for the Canal protocol, TiCDC only supports the JSON output format, while the protobuf format is not officially supported yet. To enable Canal output, specify `protocol` as `canal-json` in the `--sink-uri` configuration. For example:
```

**Review by hfxsd — APPROVED** [review]

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**